### PR TITLE
Relax version requirement to `requests`

### PR DESCRIPTION
**Pull request by tslmy**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 setup(
     name="imhotep",
-    version="3.0.0",
+    version="3.0.1",
     packages=find_packages(),
-    url="https://github.com/justinabrahms/imhotep",
+    url="https://github.com/eBay/imhotep",
```
Comment (justinabrahms) [inline setup.py]: Please revert this. The package is mine, even if eBay has a fork.

File: setup.py [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
     license="MIT",
     author="Justin Abrahms",
     author_email="justin@abrah.ms",
     description="A tool to pipe linters into code review",
     python_requires=">=3.9",
     install_requires=[
-        "requests~=2.20.0",
+        "requests",
```
Comment (justinabrahms) [inline setup.py:14]: What's the thinking here?

**Review by justinabrahms — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (tslmy) [reply setup.py]: Ah. My apologies. I misunderstood your intention.

File: setup.py (hunk 2 above)
Comment (tslmy) [reply setup.py:14]: We’re using a very basic functionality of `requests`. Requiring too strict a version makes new plugins difficult to develop. (For example, when I was developing my `imhotep_pmd`, I wanted to use a new version of this package, and there was a conflict.)

**Review by justinabrahms — APPROVED** [review]

**Pull request merged**